_recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Creamy Vegan Gnocchi With Garlic and Kale
+category: Dinners
+tags: 
+  - hearty
+  - pasta
+  - vegan
+prepTime:
+  - PT5M
+cookTime:
+  - PT7M
+recipeYield: 3 meals
+recipeCuisine: Italian
+recipeIngredient:
+  - slug: olive oil
+    amount:
+      - 1 tbs
+  - slug: garlic
+    amount: 6 cloves
+  - slug: kale
+    amount: 1 bunch
+  - slug: cashews
+    amount:
+      - 75 g
+      - 1/2 cup
+  - slug: water
+    amount:
+      - 60 ml
+      - 1/2 cup
+  - slug: gnocchi
+    amount:
+      - 500 g
+  - slug: salt
+    amount:
+      - 1/4 tps
+  - slug: freshly ground pepper
+    amount: to taste
+  - slug: chili flakes
```

Comment:
The slugs are used to link to pages about ingredients (which show you what other recipes use a certain ingredient). As such they should be URL slugs,

```suggestion
  - slug: chili-flakes
```